### PR TITLE
Fix rst docs

### PR DESCRIPTION
**Pull request by AnjoMan**
turns out PyPi doesn't like it when two links have the same text

### CODE REVIEW

**Comment by AnjoMan** [comment]
i tested it here https://test.pypi.org/project/corsons/

File: README.md [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+class Line:
+   gmr: {
+       'A': geometric_mean_radius_A
+       ...
+   }
+   r: {
+        'A': per-length resistance of conductor A in ohms
+        ...
+   }
+   phase_positions: {
```
[the remark this answers is not in the document]
Comment (AnjoMan) [reply README.md]: Good catch. I've made the change

File: README.md [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+class Line:
+   resistance: {
+       'A': per-length resistance of conductor A in ohms
+       ...
+   }
+   geometric_mean_radius: {
+       'A': geometric_mean_radius_A
+       ...
+   }
+   phase_positions: {
```
[the remark this answers is not in the document]
Comment (AnjoMan) [reply README.md]: Good catch!

File: README.md [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+
+```python
+z_primitive = CarsonsEquations(Line()).build_z_primitive()
+```
+
+For examples of how to use the model, see the [overhead wire
+tests](https://github.com/opusonesolutions/carsons/blob/master/tests/test_carsons.py).
+
+`carsons` is tested against several cable configurations from the [IEEE
+4-bus test network](http://sites.ieee.org/pes-testfeeders/resources/).
```
Comment (EmilyYLMa) [inline README.md]: Should this just be IEEE networks? Should we also mention that we tested against OpenDSS as well?

Comment (AnjoMan) [reply README.md]: Good idea. I made it just reference 'ieee test feeders' and added a note about opendss documentation as well as a link in the references

**Pull request merged**